recipes/sqlite3/all/conanfile.py

```diff
@@ -3,7 +3,7 @@
 import os
 import textwrap
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.36.0"
```

Comment:
Why incrementing the required conan version ? the only change in the recipe is the addition of calls to `set_property`, so it should not change the required conan version, because the recipe still has `.filenames` and `.names`

Reply:
:+1: